heartbeat/monitors/active/http/http_test.go

```diff
@@ -32,6 +32,8 @@ import (
 	"testing"
 	"time"
 
+	"github.com/elastic/beats/v7/heartbeat/monitors/stdfields"
+
```

Comment:
Import seems out of order, should be with the other `github.com/elastic/beats/v7/...`